constraint/pkg/client/client.go

```diff
@@ -571,23 +573,16 @@ func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.Qu
 }
 
 func (c *Client) review(ctx context.Context, target string, constraints []*unstructured.Unstructured, review interface{}, opts ...drivers.QueryOpt) (*types.Response, error) {
-	var results []*types.Result
-	var tracesBuilder strings.Builder
-
-	results, trace, err := c.driver.Query(ctx, target, constraints, review, opts...)
-	if err != nil {
-		return nil, err
-	}
-
-	if trace != nil {
-		tracesBuilder.WriteString(*trace)
-		tracesBuilder.WriteString("\n\n")
+	qr := c.driver.Query(ctx, target, constraints, review, opts...)
+	if qr.Err != nil {
+		return nil, qr.Err
 	}
 
 	return &types.Response{
-		Trace:   trace,
-		Target:  target,
-		Results: results,
+		Trace:        qr.Trace,
+		Target:       target,
+		Results:      qr.Results,
+		StatsEntries: qr.StatsEntries,
```

Comment:
move `StatsEntries` to the root and add a label of `target` in a subsequent coalition in `Review()`; have labels be namespace-d so client labels don't get clobbered